python-adapter/pom.xml

```diff
@@ -47,6 +47,11 @@
             <artifactId>jts2geojson</artifactId>
             <version>${jts2geojson.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.apache.sedona</groupId>
+            <artifactId>sedona-common</artifactId>
+            <version>${project.version}</version>
+        </dependency>
```

Comment:
Figured out I needed to add this here, but I've never figured out where the fat jar behavior comes from